src/Infiltration/ui/GameTimer.tsx

```diff
@@ -24,10 +25,14 @@ export function GameTimer({ endTimestamp }: GameTimerProps): React.ReactElement
     const ele = ref.current?.firstElementChild;
     const startTimestamp = changeRef.current.startTimestamp;
     if (!ele) return;
-    // The delay will be negative. This is because the animation starts
-    // partway completed, due to the time taken to invoke the effect.
     ele.animate([{ transform: "translateX(0%)" }, { transform: "translateX(-100%)" }], {
-      duration: endTimestamp - startTimestamp,
+      /**
+       * If duration is negative, ele.animate will throw "TypeError: duration must be non-negative or auto". It may
```

Comment:
Minor nit: Let's keep the same comment style here. `/**` is associated with public api documentation by me at least, and I use `//` for implementation comments. But it doesn't really matter, as long as there's some consistency.